lib/puck/dependency_resolver.rb

```diff
@@ -41,7 +42,6 @@ def contained_bundler(gem_home, gemfile, lockfile, groups)
         begin
           line = __LINE__ + 1 # as __LINE__ represents next statement line i JRuby, and that becomes difficult to offset
           unit = scripting_container.parse(StringIO.new(<<-"EOS").to_inputstream, __FILE__, line)
-            require 'stringio'
```

Comment:
I think `StringIO` is needed in both places (since here it's run from another ScriptingContainer), but weirdly the tests pass like this too (they passed before too, though).